packages/skuba-dive/tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "incremental": true,
+    "lib": ["ES2020"],
+    "module": "CommonJS",
+    "moduleResolution": "node",
+    "outDir": "lib",
+    "removeComments": false,
+    "target": "ES2020"
+  },
+  "exclude": ["lib*/**/*", "template/**/*"],
```

Not important, but I'm guessing we can do without this:

```suggestion
  "exclude": ["lib*/**/*"],
```